src/types/Block.ts

```diff
@@ -19,7 +19,7 @@ export type Block = {
 		((params: Params) => Promise<{items: Array<string>, max?: number, min?: number, sendOutput?: boolean, reusable?: boolean}>);
 	component?: JSX.Element | void | ((params: Params) => JSX.Element | void) |
 	((params: Params) => Promise<JSX.Element | void>);
-	chatDisabled?: boolean | ((params: Params) => boolean) | ((params: Params) => Promise<boolean>);
+	chatDisabled?: boolean | ((params: Params) => boolean) | ((params: Params) => Promise<boolean>) | null;
```

Comment:
Is there a specific reason this was added? For `chatDisabled`, it should only take in a boolean value or a function that returns a boolean value.